external/kafka/src/test/scala/org/apache/spark/streaming/kafka/KafkaStreamSuite.scala

```diff
@@ -53,8 +53,8 @@ abstract class KafkaStreamSuiteBase extends FunSuite with Eventually with Loggin
   private val zkConnectionTimeout = 6000
   private val zkSessionTimeout = 6000
   private var zookeeper: EmbeddedZookeeper = _
-  private val brokerHost = "localhost"
-  private var brokerPort = 9092
+  protected val brokerHost = "localhost"
+  protected var brokerPort = 9092
```

Comment:
Notice that in the previous PR i refactored to put brokerAddress behind the ready flag? I had done this to make sure we dont accidentally use the port value before the broker is ready. This is because the port number can change depending on which port does the broker setup successfully bind to. If you really need the port, then could you put this behind the ready flag as well?